apps/server/src/modules/rocketchat-user/entity/rocket-chat-user.entity.ts

```diff
@@ -13,7 +13,7 @@ export interface RocketChatUserEntityProps {
 	updatedAt?: Date;
 }
 
-@Entity({ tableName: 'rocketchatuser' })
+@Entity({ tableName: 'rocketchatusers' })
```

Comment:
You can not change it. If the prod DB is called rocktchatuser that this is the correct name. Otherwise you must write a DB migration script. But this should not be mixed with a other PR/Ticket.

Reply:
@WojciechGrancow

Reply:
I check the prod instance. The plural is correct. Is this entity is added by KNL?